about.html

```diff
@@ -0,0 +1,162 @@
+                </div>
+               
+            </div>
+            <ul class="menu-list">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="#">Events</a></li>
+                <li><a href="#">Documentation</a></li>
+                <li><a href="about.html">About</a></li>
+            </ul>
+    
+        </div>
+
+        <div class="navbar-desktop invisible"></div>
+    </nav>
+    <main class="main about-main">
+        <section class="headline">
+            <div class="headline-opacity">
+                <header class="headline-header">
+                    <h1>Africa's Talking Open Hackathon: FinTech</h1>
+                </header>
+                <div class="headline-desc">
+                    <p>
+                        Join us at our open hackathon every last Thursday of the month.
+                         Come build with us, present feature request, give feedback or 
+                         just contribute to our open source API's and SDKs. 
+                         Africa's Talking community allows developers to learn skills for the modern-day African Developer. 
+                         We are language and framework agnostic. All developers are welcome    
+                    </p>
+                </div>
+                <div class="when-where">
+                    <div class="when">
+                        <div class="when-header">
+                            <a class="fa far fa-calendar" href="#"></a>
+                            <h3> When</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Thursday, Jun 30 2022<br>
+                                9:00 AM - 5:30 PM (EAT)
+                            </p>
+                        </div>
+                    </div>
+                    <div class="when">
+                        <div class="when-header">
+                            <a class="fa fa-map-marker" href="#"></a>
+                            <h3> Where</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Africa's Talking HQ<br>
+                                Mzima Springs Rd, Lavington, Nairobi<br>
+                                Nairobi, 00100<br>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <div class="about-desc">
+            <h3>Africa's Talking</h3>
+            <p>
+                powers communications solutions across Africa with simplified 
+                access to telco infrastructure, developers use our powerful SMS, 
+                USSD, Voice, Airtime, and Payments APIs to bring their ideas to life, 
+                as they build and sustain scalable businesses.
+            </p>
+        </div>
```

Comment:
- [ ] This section (middle section) should include a section header, a divider, a paragraph of text, and an image or large logo, kindly consider adding a separator after the section title and an image or a large log so your design is created like the original design

|Your design|CC design|
|-|-|
|![Screenshot_20220623_201215](https://user-images.githubusercontent.com/25227022/175378917-9ab6b9e2-d26e-4635-acc0-84cfd4e3e0f2.png)|![Screenshot_20220623_201233](https://user-images.githubusercontent.com/25227022/175378930-f4a99ce3-c801-4615-ad40-304ba2ddbbbf.png)|